iron-flex-layout.html

```diff
@@ -259,6 +265,43 @@
       align-self: stretch;
     };
 
+    --layout-self-baseline: {
+      -ms-align-self: baseline;
+      /* currently not supported in safari */
+      -webkit-align-self: baseline;
+      align-self: baseline;
+    };
+
+    /* multi-line alignment in main axis */
+
+    /* currently not supported in ie */
+    /* currently not supported in safari */
```

Comment:
Sure, I will just leave the first baseline. Shall I simply suppress or comment for future addition the other definitions once they will be supported? So that someone looking for them, won't raise the same issue again?